react/react1/week1/homework/my-app/src/components/TodoList.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { TodoItem } from "./TodoItem";
+
+// functional Component
+
+export const TodoList = ({ todos }) => {
+  return (
+    <div className="lists">
+      <ul>
+        {todos.map((todo, index) => (

```

Comment:
Again, I prefer not to inline the map operations in the JSX. Assign it to a const before the return and give it a meaningful name